### PR TITLE
Fix PEP 257 docstrings violations

### PR DESCRIPTION
**Pull request by z4y4ts**
[Docstrings code style](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#docstrings):
- Follow PEP 257.
- Write docstrings for all modules, classes, and functions.
- Always format docstrings using the multi-line convention, even if there’s only one line of content (see below).
- Use three double-quotes for all docstrings.
- Start with a one-line summary. If you can’t fit a summary in one line, think harder, or refactor the code.
- Write in Sphinx-friendly prose style. Put backquotes around code names (variables, parameters, methods, etc).
- Use imperative mode.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -15,6 +15,7 @@ quality: quality-py quality-js
 quality-py:
 	pep8 . --format=pylint --max-line-length=120
 	pylint -f colorized video_xblock
+	pydocstyle -e --add-ignore=D212,D203
```
Comment (z4y4ts) [inline Makefile]: Move to `.pydocstyle` with comments briefly stating a reason for ignore.

File: video_xblock/__init__.py [hunk 2]
```diff
@@ -1,5 +1,3 @@
-"""
-Video xblock module
-"""
+"""Video xblock module."""
```
Comment (z4y4ts) [inline video_xblock/__init__.py]: > Always format docstrings using the multi-line convention, even if there’s only one line of content.

https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#docstrings

File: video_xblock/backends/base.py [hunk 3]
```diff
@@ -23,28 +24,24 @@ class BaseApiClient(object):
...
-
     """
 
     @abc.abstractmethod
     def get(self, url, headers=None, can_retry=True):
         """
         Issue REST GET request to a given URL.
 
-        Can throw ApiClientError or it's subclass.
+        Can throw `ApiClientError` or it's subclass.
```
Comment (z4y4ts) [inline video_xblock/backends/base.py]: :+1:

File: video_xblock/backends/brightcove.py [hunk 4]
```diff
@@ -204,15 +223,14 @@ def upload_ingest_profile(self, account_id, ingest_profile):
 
     def submit_retranscode_job(self, account_id, video_id, profile_type):
         """
-        Draft version.
-        Submit video for re-transcoding via Brightcove's Dynamic Ingestion API.
+        Draft version. Submit video for re-transcoding via Brightcove's Dynamic Ingestion API.
```
Comment (z4y4ts) [inline video_xblock/backends/brightcove.py]: Well, now it's not a "Draft version" anymore.

File: video_xblock/backends/wistia.py [hunk 5]
```diff
@@ -209,10 +206,7 @@ def get_default_transcripts(self, **kwargs):
 
     @staticmethod
     def format_transcript_text_line(line):
-        """
-        Replaces comma with dot in timings, e.g. 00:00:10,500 should be 00:00:10.500
-
-        """
+        """Replace comma with dot in timings, e.g. 00:00:10,500 should be 00:00:10.500."""
         pattern = re.compile(r"\d{2}:\d{2}:\d{2},\d{3}")
```
Comment (z4y4ts) [inline video_xblock/backends/wistia.py]: This is out of scope of this PR. But why this method can't be simplified down to something like this?
```py
return re.sub(r'(\d{2}:\d{2}),(\d{3})', r'\1.\2', line)
```

File: video_xblock/settings.py [hunk 6]
```diff
@@ -1,10 +1,10 @@
 # -*- coding: utf-8 -*-
 """
-This is the common settings file, intended to set defaults
-"""
+Common settings file, intended to set defaults.
 
-# Source:
-#  https://github.com/edx/edx-platform/blob/master/lms/envs/common.py#L2457
+Source:
+  https://github.com/edx/edx-platform/blob/master/lms/envs/common.py#L2457
```
Comment (z4y4ts) [inline video_xblock/settings.py]: It's better to be https://github.com/edx/edx-platform/blob/open-release/eucalyptus.master/lms/envs/common.py#L2390
Since master is always moving.

File: video_xblock/backends/wistia.py (hunk 5 above)
Comment (OPersian) [inline video_xblock/backends/wistia.py]: Thanks! Addressed in PR #118

File: video_xblock/video_xblock.py [hunk 7]
```diff
@@ -552,31 +597,36 @@ def _make_field_info(self, field_name, field):
...
         """
         if file_field:
             return os.path.join('/', file_field)
         return ''
 
     def get_transcript_download_link(self):
-        """
-        Returns link for downloading transcript of the current captions language if it exists
-        """
+        """Return link for downloading of a transcript of the current captions' language (if a transcript exists)."""
```
Comment (z4y4ts) [inline video_xblock/video_xblock.py]: Please make it multiline.

File: video_xblock/video_xblock.py [hunk 8]
```diff
@@ -618,10 +674,14 @@ def ui_dispatch(self, _request, suffix):
         """
         Dispatcher for a requests sent by dynamic Front-end components.
 
-        Typical use case: Front-end wants to check with backend if it's ok to show
-        certain part of UI.
-        """
+        Typical use case: Front-end wants to check with backend if it's ok to show certain part of UI.
 
+        Arguments:
+            _request (xblock.django.request.DjangoWebobRequest): Incoming request data (if any).
```
Comment (z4y4ts) [inline video_xblock/video_xblock.py]: Please add a note  `Not used`

Comment (OPersian) [inline video_xblock/video_xblock.py]: Done.

**Comment by z4y4ts** [comment]
Good to go 👍 

GitHub doesn't let me approve this PR, since I've opened it. @dorosh @xahgmah @TamoshaytisV please take a look.
